modules/ztp-provisioning-lvm-storage.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-advanced-policy-config.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-provisioning-lvm-storage_{context}"]
+= Configuring {lvms} using PolicyGenTemplate CRs
+
+You can configure {lvms-first} for managed clusters that you deploy with {ztp-first}.
+
+[NOTE]
+====
+You use {lvms} to persist event subscriptions when you use PTP events or bare-metal hardware events with HTTP transport.
+====
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+* Create a Git repository where you manage your custom site configuration data.
+
+.Procedure
+
+. To configure {lvms} for new managed clusters, add the following YAML to `spec.sourceFiles` in the `common-ranGen.yaml` file:
++
+[source,yaml,subs="attributes+"]
+----
+- fileName: StorageLVMOSubscriptionNS.yaml
+  policyName: subscription-policies
+- fileName: StorageLVMOSubscriptionOperGroup.yaml
+  policyName: subscription-policies
+- fileName: StorageLVMOSubscription.yaml
+  spec:
+    name: lvms-operator
+    channel: stable-4.12
+    source: redhat-operator-index
```

Comment:
Why this source? the default catalogsource for LVMO is redhat-operators.

There is no need to include a source since it is set to default's redhat-operators in the source-cr : https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/StorageLVMOSubscription.yaml#L11

You need to include a different source if you're creating a custom catalogsource.